doc/architecture.md

```diff
@@ -10,10 +10,10 @@ The architecture allows for multiple `EARTH_COMP` instances.
 NEMS includes atmosphere, ocean, ice, wave, land,
 aerosol/chemistry, and hydrologic models, with coupling interface and
 utilities based on the 
-[Earth System Modeling Framework (ESMF)](https://www.earthsystemcog.org/projects/esmf/).
+[Earth System Modeling Framework (ESMF)](https://www.earthsystemmodeling.org).
```

Comment:
I see the website is redirected https://www.icams-portal.gov/, it may need to be updated later

Reply:
I think this site works without remodeling...
https://www.earthsystemmodeling.org